.github/workflows/update-pr-ready-review.yml

```diff
@@ -0,0 +1,87 @@
+name: Move card to Need Dev Review project column
+on:
+  pull_request_target:
+   types:
+     - ready_for_review
+     - opened
+     - reopened
+
+jobs:
+  move_card_to_need_dev_review:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Project Data
+        env:
+          GITHUB_TOKEN: ${{ secrets.BOOSTED_MOD_PERSONAL_TOKEN_CLASSIC }}
+          ORGANIZATION: ${{ github.repository_owner }}
+          PR_ID: ${{ github.event.pull_request.node_id }}
+          PROJECT_NUMBER: 12
```

Comment:
For `PROJECT_NUMBER` and `PROJECT_TARGET_COL`, we could use Actions variables in order to ease the maintenance if the project is duplicated, the column name changes, the tech lead name changes (other PR), etc.

Reply:
```suggestion
          PROJECT_NUMBER: ${{ vars.PR_BOARD_PROJECT_NUMBER }}
```